pkg/models/forum/comment.go

```diff
@@ -75,21 +88,39 @@ func (f *ForumStore) RateComment(commentID, userID int) (bool, error) {
 	return rateOk, err
 }
 
-func (f *ForumStore) GetCommentRatings(commentID int) (likes []int64, err error) {
+func (f *ForumStore) getCommentLikers(commentID int) (likersIDs []int64, err error) {
 	rows, err := f.DB.Query(`SELECT user_id FROM ratings WHERE comment_id = $1;`, commentID)
 	if err != nil {
 		err = fmt.Errorf("Can't read rating-rows from db: %v", err)
 	}
 	defer rows.Close()
 
 	for rows.Next() {
-		var like int64
-		err = rows.Scan(&like)
+		var likerID int64
+		err = rows.Scan(&likerID)
 		if err != nil {
```

Comment:
Why is there no return statement? Why do application need to continue work in this function, if error happens?